src/gui/guiFormSpecMenu.cpp

```diff
@@ -825,12 +825,23 @@ void GUIFormSpecMenu::parseImage(parserData* data, const std::string &element)
 	core::rect<s32> middle;
 	if (parts.size() >= 4)
 		parseMiddleRect(parts[3], &middle);
-
-	GUIAnimatedImage *e = new GUIAnimatedImage(Environment, data->current_parent,
-		spec.fid, rect);
-
-	e->setTexture(texture);
-	e->setMiddleRect(middle);
+	
+	gui::IGUIElement *e;
+	if (middle.getArea() > 0) {
```

Comment:
would be good to add a comment to indicate this is a bug workaround and not just "normally there"

Reply:
Added an explanation.